src/libsyntax_pos/symbol.rs

```diff
@@ -318,7 +318,6 @@ declare_keywords! {
     (46, Offsetof,           "offsetof")
     (47, Override,           "override")
     (48, Priv,               "priv")
-    (49, Proc,               "proc")
```

Comment:
The keyword list shouldn't contain holes (otherwise random identifiers will be interpreted as keywords), so the numbers of following keywords have to be shifted by one.

Reply:
Ah, good catch. @zesterer?

(Also, I wonder if we could construct this list in another way so that it auto-numbers, eliminating this issue in the future.)

Reply:
My experience with the compiler codebase is less than excellent. Is this the sort of thing that could be easily macro-driven?

Reply:
This is the only keyword removed in the last 3-4 years, certainly not something that need to be automated by extra macro machinery.

Reply:
@petrochenkov  I may be over-thinking this. Is this literally just a case of enumerating everything that comes after the removed keyword?

Reply:
@zesterer 
Yes